Workaround for v3.7 dockercfg secret issue

jcpowermac opened the pull request:
This PR includes the ability to download the origin v3.6.1 which
will correctly create a dockercfg secret.

codificat commented on ansible/group_vars/all.yml.example:
```diff
@@ -36,8 +36,22 @@ notify_email_replyto: noreply@example.com
 # OpenShift client command options
 # --------------------------------
+# Path to openshift client
+oc_path: "/usr/bin/oc"
```
Well... this default can break some setups (like mine... I have ~/bin/oc symlinked to some version for easy switching). Can we make the default detect the current `oc` path?

codificat reviewed (commented):
A couple of minor comments but otherwise LGTM

codificat commented on ansible/main.yml:
```diff
@@ -1,8 +1,30 @@
+        - name: Extract oc
+          unarchive:
+            src: /tmp/oc-client.tar
+            dest: /tmp/
```
Talking about site-specific breakage, isn't `/tmp` risky? noexec configurations, strange platforms, ... just wondering

jcpowermac replied:
@codificat I am good to put it anywhere.  Somewhere else you think?

jcpowermac commented:
@codificat added your suggestion of find `oc` binary.  Also added `no_log` as suggested by @aweiteka

jcpowermac committed:
Workaround for v3.7 dockercfg secret issue

This PR includes the ability to download the origin v3.6.1 which
will correctly create a dockercfg secret.

Added `no_log` to remove the displaying of the tokens
Added suggestion from @codificat to locate `oc` binary.
Added check for file matching if 0 use workaround.
